docs/_media/delayed.md

```diff
@@ -0,0 +1,7 @@
+- This is from the `delayed.md`
+
+```bash
+sleep 1000
+echo delayed
+exit 0
+```
```

Comment:
not sure why these files ?

can we add a note that this is just an example ?

Reply:
I needed 2 files that will occupy a lot of space so that it's clear that the order of emmbedding is wrong...